build.sbt

```diff
@@ -101,12 +101,20 @@ lazy val root = Project("spark-redshift", file("."))
      * Release settings *
      ********************/
 
+    publishTo := {
+      val nexus = "https://oss.sonatype.org/"
```

Comment:
Please note that you need to specify also Nexus username and password.
Those are completely unrelated with the GPG credentials.

Reply:
Oh yes I put those in a file offline on disk (not in the repo).
I created sonatype.sbt following https://www.scala-sbt.org/0.13/docs/Using-Sonatype.html
and I also created a .sbt/.credentials file following: https://www.scala-sbt.org/1.x/docs/Publishing.html#Credentials

see line https://github.com/spark-redshift-community/spark-redshift/pull/28/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR60

I hope that's enough?